lemminx-maven/src/main/java/org/eclipse/lemminx/maven/DiagnosticRequest.java

```diff
@@ -62,6 +83,16 @@ public Range getRange() {
 				((DOMElement) node).getEndTagOpenOffset(), xmlDocument);
 	}
 
+	public void addDiagnostic(String errorMessage, DiagnosticSeverity severity) {
+		Diagnostic diagnostic = new Diagnostic(this.getRange(), errorMessage, severity,
+				this.getXMLDocument().getDocumentURI(), "XML");
+
+		// TODO: Should diagnostics just be a set?
```

Comment:
Yes, probably. Especially since they're actually not ordered.

Reply:
Sounds good, will do.